src/test/java/teammates/test/pageobjects/InstructorStudentListPage.java

```diff
@@ -29,6 +38,31 @@ public InstructorStudentListPage(Browser browser) {
     protected boolean containsExpectedPageContents() {
         return getPageSource().contains("<h1>Student List</h1>");
     }
+    
+    public void toggleShowEmailCheckbox() {
+        click(showEmailCheckbox);
+    }
+    
+    public void clickCopyEmailButton() {
+        click(copyEmailButton);
+    }
+    
+    public boolean isCopyEmailButtonVisible() {
+        return copyEmailButton.isDisplayed();
+    }
+    
+    public boolean isCopyEmailPopoverVisible() {
+        String xpath = "//button[@id='copy_email_button']/following-sibling::div[contains(@class, 'popover')]";
```

Comment:
[Here's](https://saucelabs.com/resources/articles/selenium-tips-css-selectors) a guide on css selectors. You can use `#copy_email_button > div.popover`, which should work.